config/locales/en.yml

```diff
@@ -910,6 +938,9 @@ en:
     recent_updates_summary: Out of the last %{total} updates, %{failure_count} failed and %{error_count} encountered errors.
     revisions: Recent Edits
     references_count: References Added
+    references_doc_separated:
```

Comment:
I missed these `separated` message keys earlier.

I think it's better to avoid renaming (or duplicating) existing messages, which will cause i18n regressions for other languages unless/until translators add the translations for the new keys.

I suggest, instead, adding the new message key like `references_doc_wikidata`, then use the `isWikidata` logic or similar to append to the key name if appropriate.